pinot-spi/src/main/java/org/apache/pinot/spi/env/PinotConfiguration.java

```diff
@@ -354,7 +356,12 @@ public long getProperty(String name, long defaultValue) {
    * @return the property String value. Fallback to default value if missing.
    */
   public String getProperty(String name, String defaultValue) {
-    return getRawProperty(name, defaultValue).toString();
+    Object rawProperty = getRawProperty(name, defaultValue);
```

Comment:
The behavior is not aligned with the `getProperty(String name)`. I feel we should make them consistent to prevent unexpected behavior. You may sync with @daniellavoie who contributed this file

Reply:
sounds good. although I think it should be addressed in separate PR as TUNER_PACKAGE also uses this behavior. will sync with @daniellavoie